### PR TITLE
[NO-ISSUE] refactor: readding data streaming and edge pulse texts

### PR DESCRIPTION
**Pull request by hannah-sk**
Modifications:

* Added correct Data Streaming labels, descriptions, placeholders, helper texts.
* Added Edge Pulse correct text for Pre-loading Tag tab.

### CODE REVIEW

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 1]
```diff
@@ -45,8 +49,7 @@
             class="w-full"
           />
           <small class="text-color-secondary text-xs font-normal leading-tight">
-            Data Source is the Azion Platform that generates the events from where you want to
-            collect data.
+            Represents the data source your data will be collected from.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:52]:
```suggestion
            Represents the data source the data will be collected from.
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 2]
```diff
@@ -85,15 +87,14 @@
           class="min-h-[100px] surface-border border rounded-sm overflow-hidden"
         />
         <small class="text-color-secondary text-xs font-normal leading-tight">
-          Data Set is a format chosen to send the data to your endpoint. It must be a valid JSON
-          format. The requests are separated from each other by \n character.
+          Exhibits or allows you to write the variables that'll be sent to your connector in a JSON format.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:90]:
```suggestion
          Exhibits or allows writing the variables that'll be sent to the connector in a JSON format.
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 3]
```diff
@@ -85,15 +87,14 @@
...
         </small>
       </div>
     </template>
   </FormHorizontal>
   <FormHorizontal
     title="Domains"
-    description="Description"
+    description="Exhibits or allows you to write the variables that'll be sent to your connector in a JSON format."
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:97]:
```suggestion
    description="Exhibits or allows writing the variables that'll be sent to the connector in a JSON format."
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 4]
```diff
@@ -120,7 +121,7 @@
             <label class="text-color text-sm font-normal leading-tight">Filter Domains </label>
           </div>
           <small class="text-color-secondary text-xs font-normal leading-tight"
-            >In case you select the All Domains option, you can activate the Sampling option</small
+            >In case you select the All Current and Future option, you can activate the Sampling option.</small
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:124]:
```suggestion
            >By selecting the All Current and Future option, you can activate the Sampling option.</small
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 5]
```diff
@@ -222,20 +223,21 @@
...
+          After activating and saving these settings, all other streams will become
           inactive.
         </InlineMessage>
       </div>
     </template>
   </FormHorizontal>
 
-  <FormHorizontal title="Destination">
+  <FormHorizontal title="Destination"
+            description="Decide to which data platform your collected data will be sent to. All fields must be filled in with information provided by the third-party platform.">
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:234]:
```suggestion
            description="Define to which data platform your collected data will be sent to. All fields must be filled in with information provided by the third-party platform.">
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 6]
```diff
@@ -468,9 +475,10 @@
             :class="{ 'p-invalid': accessKeyError }"
             :feedback="false"
             toggleMask
+            placeholder="ORIA5ZEH9MW4NL5OITY4"
           />
           <small class="text-color-secondary text-sm font-normal leading-tight">
-            The access key of your S3 bucket.
+            Public key to access your bucket.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:481]:
```suggestion
            Public key to access the bucket.
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 7]
```diff
@@ -489,13 +497,14 @@
...
             type="text"
+            placeholder="+PLjkUWJyOLth3anuWXcLLVrMLeiiiThIokaPEiw"
             class="flex flex-col w-full"
             :class="{ 'p-invalid': secretKeyError }"
             :feedback="false"
             toggleMask
           />
           <small class="text-color-secondary text-sm font-normal leading-tight">
-            The secret key of your S3 bucket.
+            Secret key to access your bucket.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:507]:
```suggestion
            Secret key to access the bucket.
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 8]
```diff
@@ -514,13 +523,14 @@
...
             type="text"
+            placeholder="waf_logs_1622575860091_37d66e78-c308-4006-9d4d-1c013ed89276"
             class="flex flex-col w-full"
             :class="{ 'p-invalid': objectKeyError }"
             :feedback="false"
             toggleMask
           />
           <small class="text-color-secondary text-sm font-normal leading-tight">
-            The format of the object that will be created in your S3 bucket.
+            Prefix added to the name of the uploaded object to appear on the files that will be sent. Composed of Prefix + Timestamp + UUID.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:533]:
```suggestion
            Prefix added to the name of the uploaded object to appear on the files that'll be sent. Composed of Prefix + Timestamp + UUID.
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 9]
```diff
@@ -715,8 +728,7 @@
             :class="{ 'p-invalid': splunkUrlError }"
           />
           <small class="text-color-secondary text-sm font-normal leading-tight">
-            The URL that will receive the collected data from Data Streaming, if you have an
-            alternative index to point, you can do at the end of the URL.
+            URL that'll receive the collected data. If you have an alternative index to point, add it at the end of the URL.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:731]:
```suggestion
            URL that'll receive the collected data. Having an alternative index to point, add it at the end of the URL.
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 10]
```diff
@@ -778,15 +792,16 @@
...
           >
           <InputText
             v-model="awsRegion"
             type="text"
+            placeholder="us-east-1"
             :class="{ 'p-invalid': awsRegionError }"
           />
           <small class="text-color-secondary text-sm font-normal leading-tight">
-            The region of your S3 bucket.
+            Region where your Amazon Kinesis instance is running.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:804]:
```suggestion
            Region where the Amazon Kinesis instance is running.
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 11]
```diff
@@ -1095,15 +1116,15 @@
   </FormHorizontal>
   <FormHorizontal
     title="Payload"
-    description="Description"
+    description="Customize the essential information that will be sent in your data."
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:1119]:
```suggestion
    description="Customize the essential information that'll be sent in the data."
```

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue [hunk 12]
```diff
@@ -1112,8 +1133,7 @@
           :class="{ 'p-invalid': payloadFormatError }"
         />
         <small class="text-color-secondary text-xs font-normal leading-tight">
-          The format that payload will be sent. The $dataset variable will be replaced by all logs
-          already with the log line separator applied.
+          Character that will be used at the end of each log line. The "\n" escape sequence breaks values into different lines in NDJSON format.
```
Comment (MarianaAguilera) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:1136]:
```suggestion
          Character that'll be used at the end of each log line. The "\n" escape sequence breaks values into different lines in NDJSON format.
```

**Review by MarianaAguilera — COMMENTED** [review]
left some suggestions.

File: src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue (hunk 3 above)
Comment (hannah-sk) [inline src/views/DataStreaming/FormFields/FormFieldsDataStreaming.vue:97]:
```suggestion
    description="Associate domains with this stream to define the addresses from which the data will be collected."
```

**Review by bru-andrade — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

**Review by csfeijo — APPROVED** [review]

**Pull request merged**